api/src/pcapi/alembic/versions/20240124T150656_6be778edb2f4_drop_unused_columns_from_product.py

```diff
@@ -0,0 +1,32 @@
+"""Drop "isNational", "url", "owningOffererId" & "isSynchronizationCompatible" column to "offer" table."""
+from alembic import op
+import sqlalchemy as sa
+
+
+# pre/post deployment: post
+# revision identifiers, used by Alembic.
+revision = "6be778edb2f4"
+down_revision = "6993f48deb4b"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.drop_column("product", "isNational")
+    op.drop_column("product", "url")
+    op.drop_column("product", "owningOffererId")
+    op.drop_column("product", "isSynchronizationCompatible")
+
+
+def downgrade() -> None:
+    op.add_column(
+        "product",
+        sa.Column("isSynchronizationCompatible", sa.BOOLEAN(), server_default=sa.text("true"), nullable=False),
+    )
+    op.add_column("product", sa.Column("owningOffererId", sa.INTEGER(), nullable=True))
+    op.add_column("product", sa.Column("url", sa.VARCHAR(length=255), nullable=True))
+    op.add_column(
+        "product",
+        sa.Column("isNational", sa.BOOLEAN(), server_default=sa.text("false"), nullable=False),
+    )
+    op.create_foreign_key("product_owning_offerer_id_fkey", "product", "offerer", ["owningOffererId"], ["id"])
```

Comment:
De même en cas de rollback on aura du mal à faire passer ces migrations, donc on préférera ajouter des clauses `IF NOT EXISTS`.